apps/code/python_toolbox.cpp

```diff
@@ -728,6 +775,24 @@ bool PythonToolbox::selectLeaf(int selectedRow, bool quitToolbox) {
   return true;
 }
 
+// This is the same function as in the Toolbox class, but we need to override it because we need to handle the Key selector differently.
+bool PythonToolbox::selectSubMenu(int selectedRow) {
+  // If the selected row is a is the Key selector, we display the IonKeySelectorViewController
+  if (m_messageTreeModel->childAtIndex(selectedRow)->label() == I18n::Message::IonSelector) {
+    m_ionKeys.setSender(sender());
+    Container::activeApp()->displayModalViewController(static_cast<ViewController*>(&m_ionKeys), 0.f, 0.f, Metric::PopUpTopMargin, Metric::PopUpLeftMargin, 0, Metric::PopUpRightMargin);
+    return true;
+  }
+  m_selectableTableView.deselectTable();
+  m_messageTreeModel = static_cast<const ToolboxMessageTree *>(m_messageTreeModel->childAtIndex(selectedRow));
+  if (m_messageTreeModel->isFork()) {
+    assert(m_messageTreeModel->numberOfChildren() < 0);
+    m_messageTreeModel = static_cast<const ToolboxMessageTree *>(m_messageTreeModel->childAtIndex(indexAfterFork()));
+  }
+  return NestedMenuController::selectSubMenu(selectedRow);
```

Comment:
Why do not call the super implementation ?

Reply:
How ?